app/controllers/flatirons/saas/subscriptions_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require_dependency 'flatirons/saas/application_controller'
```

Comment:
is there a reason to not inherit from `< Flatirons::Saas::ApplicationController`?

Reply:
It was generated by the rails framework